commands/build/lib/build.js

```diff
@@ -224,13 +227,14 @@ const systemjs = async (argv, core) => {
         const { external = [] } = cfg.systemjs || {};
         return mergeArray(defaultExternal, external);
       },
-      getOutputFile: ({ config: cfg }) => {
-        const { outputFile = 'systemjs/index.js' } = cfg.systemjs || {};
-        return outputFile;
-      },
+      getOutputFile: ({ pkg }) => pkg.systemjs,
       getOutputName: () => undefined,
+      enabled: ({ pkg }) => !!pkg.systemjs,
     },
   });
+  if (!c) {
+    return Promise.resolve();
```

Comment:
use
```suggestion
    return undefined;
```
inside an async function there is no need to create a new empty promise just returning undefined will have the same result.

Reply:
Fixed this occurrence and similar one when build ESM bundle.